### PR TITLE
chore: update SDK from api-definitions

### PR DESCRIPTION
**Pull request by ghost**
This PR is automatically generated and updates SDK based on a change in the API definitions

### CODE REVIEW

**Comment by rebilly-machine-user** [comment]
Triggered due to: Implement patch credit memo endpoint defs Rebilly/api-definitions#1046

File: src/Model/InlineObject.php [hunk 1]
```diff
@@ -1,55 +1,311 @@
...
 
 use JsonSerializable;
+use InvalidArgumentException;
+use TypeError;
 
-class InlineObject implements JsonSerializable
-{
-    private array $fields = [];
-
+class InlineObject {
```
Comment (adamaltman) [inline src/Model/InlineObject.php:11]: This is a strange name, no?

Comment (nanobash) [reply src/Model/InlineObject.php:11]: Not sure how the auto generation works, but seems that the interface was removed here. Maybe who have worked with auto generation can comment on it?

Comment (justnero) [reply src/Model/InlineObject.php:11]: It is a fallback name for objects without a separate schema name or title. PatchCreditMemo operation definition must be fixed.

**Review by justnero — DISMISSED** [review]

**Review by nanobash — DISMISSED** [review]

**Review by justnero — APPROVED** [review]

**Review by mikerebilly — APPROVED** [review]

**Pull request merged**